editor/components/block-list/style.scss

```diff
@@ -581,17 +581,27 @@
 		}
 	}
 
-	// Show side UI inline below the block on mobile
+	// Show side UI inline below the block on mobile.
 	.editor-block-list__block-mobile-toolbar {
 		display: flex;
 		flex-direction: row;
-		margin-top: $item-spacing;
+		margin-top: $item-spacing + $block-toolbar-height; // Make room for the formatting bar above. 
```

Comment:
This comment helps, but `// Make room for the height of the block toolbar above.` is even more explicit/helpful.